v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerIT.java

```diff
@@ -77,11 +75,15 @@ public class DataStreamToSpannerIT extends TemplateTestBase {
   private static final String MEMBER = "member";
   private static final String ENTRY_ADDED = "entry_added";
 
+  private String gcsPrefix;
+  private String dlqGcsPrefix;
+
   private static final List<String> COLUMNS = List.of(ROW_ID, NAME, AGE, MEMBER, ENTRY_ADDED);
 
-  private CustomMySQLResourceManager jdbcResourceManager;
+  private CloudSqlResourceManager jdbcResourceManager;
```

Comment:
Nit: We should probably rename the variable as well for clarity that this CloudSQL being used behind the scenes.

Reply:
Good point!